client/src/components/Menu/Menu.scss

```diff
@@ -402,6 +402,8 @@
   margin-top: -1px;
   left: 13px;
   font-size: $h3-font-size;
+  min-width: 16px;
+  background-repeat: no-repeat;
```

Comment:
The background position seems to be vertically off-center when testing this, shouldn’t we set it to `50% 50%`?

Reply:
Thanks Loz. I have added `background-position: 50% 2px;`. I tried 50% 50% but that dropped the icon way too low. Have a look and let me know what you think.